apps/site/src/pages/api/blocks/index.api.ts

```diff
@@ -114,6 +115,23 @@ export default createApiKeyRequiredHandler<null, ApiBlockSearchResponse>()
 
     // @todo paginate response
 
+    const clientIp = parseClientIp(req);
+
+    const userId = req.user.id;
+
+    await sendReport({
+      event: "block_listing",
```

Comment:
I think 'block listing' could be confused with 'block publishing'
```suggestion
      event: "block_search",
```

Reply:
9562bd1764d3fd40e6f7c407345cc2665faba651